gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,655 @@
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Endpoint Up",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 29
+        },
+        {
+            "add_timeframe": true,
+            "auto_refresh": false,
+            "background_color": "gray",
+            "bgcolor": "gray",
+            "content": "Service Checks",
+            "font_size": "18",
+            "height": 25,
+            "html": "Service Checks",
+            "isShared": false,
+            "refresh_every": 30000,
+            "scaleFactor": 1,
+            "show_tick": true,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "right",
+            "tick_pos": "50%",
+            "title": false,
+            "type": "note",
+            "width": 15,
+            "x": 1,
+            "y": 12
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 13,
+            "isShared": false,
+            "legend": false,
+            "markers": [],
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "markers": [],
+                "requests": [
+                    {
+                        "q": "avg:gitlab.http_requests_total{*}.as_count()",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Http requests",
+            "type": "timeseries",
+            "viz": "timeseries",
+            "width": 43,
+            "x": 34,
+            "y": 13,
+            "yaxis": {
+                "includeZero": true,
+                "label": "",
+                "max": "auto",
+                "min": "auto",
+                "scale": "linear"
+            }
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 13,
+            "isShared": false,
+            "legend": false,
+            "markers": [],
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "markers": [],
+                "requests": [
+                    {
+                        "q": "avg:gitlab.http_request_duration_seconds.count{*}.as_count()",
```

Comment:
`http_request_duration_seconds` is a distribution metric. Here you're plotting the number of request which is already in the previous graph.